css-layout-classes/styles.css

```diff
@@ -0,0 +1,33 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.row .img {
+  width: calc(100% / 2);
+}
+
+.row img {
+  width: 150px;
+}
+
+.row .image img {
+  width: 200px;
+  height: auto;
+}
```

Comment:
If we wrap all the elements (images or texts) with columns, then we wouldn't need to set width on images